common/src/main/java/net/william278/huskclaims/claim/ClaimWorld.java

```diff
@@ -46,6 +46,7 @@
 import java.util.*;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
+import java.util.logging.Level;
```

Comment:
Don't forget to remove an unused import :)

Reply:
Fixed oops :)